### ISSUE TITLE
[Enhancement] Allow workspace bulk commands

### ISSUE THREAD

**Issue by ccecvb** [ForkIssues/TrackerWin#1763]
On the command line, we use [git bulk](https://github.com/tj/git-extras/blob/master/Commands.md#git-bulk)
Can something similar be added to the workspaces in fork ?

Commands I regularly use are

* fetch
* pull
* checkout [-b]
* tag
* push [--tags]
* status

## Related second request

If the bulk feature would be implemented, I would most like still go to the command line for some operations.
Could a version of git bulk for the command line be made that uses the fork workspace definition instead of the current config
e.g `git fbulk status` would check the fork config, locate the workspace a repo is added to, error if it's found in multiple workspaces, and execute the status command on all repos in the workspace.

**Comment by ccecvb** [comment]
I just realized that maybe the second request could form the basis of what the UI version should do, except for the error, because in the UI it would be clear which workspace should be used

**Comment by DanPristupov** [comment]
> Could a version of git bulk for the command line be made that uses the fork workspace definition instead of the current config

Yes, you can make it. Fork config is located at `%localappdata%\fork\settings.json`. There is `Workspaces` section which has the workspace definitions.

**Comment by ccecvb** [comment]
I was already thinking of creating a git-fbulk based on this, thank you for the info.
